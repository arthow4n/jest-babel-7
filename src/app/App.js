import React from 'react';
import { hot } from 'react-hot-loader';
import styled from 'styled-components';

const Text = styled.div`
  width: 200px;
  margin: 0 auto;
`;

const App = () => (
  <Text>You did it!</Text>
);

export default hot(module)(App);
