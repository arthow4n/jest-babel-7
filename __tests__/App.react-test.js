import React from 'react';
import Enzyme, { mount } from 'enzyme';
import renderer from 'react-test-renderer';
import Adapter from 'enzyme-adapter-react-16';
import 'jest-styled-components';
import App from './../src/app/App.js';

Enzyme.configure({ adapter: new Adapter() });

test('Compare snapshots', () => {
  const component = renderer.create(<App />);

  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test('Renders without crashing', () => {
  mount(<App />);
});
