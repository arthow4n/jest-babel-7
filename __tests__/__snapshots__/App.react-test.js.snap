// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Compare snapshots 1`] = `
.c0 {
  width: 200px;
  margin: 0 auto;
}

<div
  className="c0"
>
  You did it!
</div>
`;
